app/app.go

```diff
@@ -80,10 +88,22 @@ func Start(ctx context.Context, cfg config.Config, env env.SpaceEnv) {
 		space.WithAddWatchFileFunc(sync.AddFileWatch),
 	)
 
+	// setup FUSE FS Handler
+	fuseHandler, err := spacefs.NewSpaceFS(ctx, fs_data_source.NewSpaceFSDataSource(sv))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	mountPath := cfg.GetString(config.FuseMountPath, "~/")
```

Comment:
cant we just hide away this bootstrap piece within a Start method thru the fuse interface we provide? seems that app.go is getting to big and we just need the `fuseController` at this level so a `New` method that sets up everything to return a fuseController will be cleaner from the app.go

Reply:
Oh true, I missed this part. The reason I created FuseController is to move most of these initializations away from app.go because it was becoming harder to understand what was going on as per initialization in that file. I'll work on moving this to FuseController also.